pkg/helm/controller/controller.go

```diff
@@ -37,22 +37,22 @@ var log = logf.Log.WithName("helm.controller")
 // WatchOptions contains the necessary values to create a new controller that
 // manages helm releases in a particular namespace based on a GVK watch.
 type WatchOptions struct {
-	Namespace      string
-	GVK            schema.GroupVersionKind
-	ManagerFactory release.ManagerFactory
-	ResyncPeriod   time.Duration
+	Namespace       string
+	GVK             schema.GroupVersionKind
+	ManagerFactory  release.ManagerFactory
+	ReconcilePeriod time.Duration
 }
 
 // Add creates a new helm operator controller and adds it to the manager
 func Add(mgr manager.Manager, options WatchOptions) error {
-	if options.ResyncPeriod == 0 {
-		options.ResyncPeriod = time.Minute
+	if options.ReconcilePeriod == 0 {
+		options.ReconcilePeriod = time.Minute
```

Comment:
Just curious any particular reason for this default?

Reply:
We wanted it to not be to long between the reconcile that things could be wrong for a long time, but also didn't want to abuse the API server. This was a best guess on balancing those concerns :)

Reply:
It's only because sometimes you want the reconcile period to be 0, 
I encountered this before in one of the projects, we wanted the loop to be triggered only when the external event triggered it, so we set it to 0. Maybe we should take that into consideration here as well? https://github.com/operator-framework/operator-sdk/issues/484#issuecomment-423466519

Reply:
We can also merge this first and since it's done in 855. Up to you. 👍